src/main/kotlin/me/spradling/gift/core/api/Application.kt

```diff
@@ -1,6 +1,7 @@
 package me.spradling.gift.core.api
```

Comment:
You shouldn't be checking in the above `.DS_Store` file.

Reply:
Removed